src/web/content/dynamic/searchmysite/templates/search/results.html

```diff
@@ -50,9 +50,11 @@
 						<div class="search-result sms-b-72">
 							<h2 class="sms-h2 sms-b-8">
 								{% if result['contains_adverts'] %}<img src="/static/images/containsadverts44x24.gif" alt="Page contains adverts" class="float-left" style="margin-right: 4px">{% endif %}
-								{% if result['published_date'] %}<span id="result-published-date">{{ result['published_date'] }}</span>:{% endif %}
```

Comment:
Moving date outside of the h2. Are there any impacts that might affect other scenarios other than dated results? Let me know and I can test.

Reply:
I don't think there's any particular reason why the date was in the h2, so should be fine to move it outside the h2.